### PR TITLE
create tinywizard bio

### PR DESCRIPTION
**Pull request by tinywizard**
:warning: Work in Progress.  Not ready to merge yet :warning: 
Adding my bio to #46

### CODE REVIEW

File: docs/tinywizard.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# I LIKE CATS
+- WHOO!
```
Comment (tinywizard) [inline docs/tinywizard.md]: Add what kind of cats

**Comment by tinywizard** [comment]
Going to add kinds of cats I like meow.  :smile_cat:

**Comment by tinywizard** [comment]
@githubteacher Would love your input on what types of cats are the best ones.

**Comment by hilaryparker** [comment]
@tinywizard great bio

**Pull request merged**